Fixes: #2460 – Add pull request template

mischah opened the pull request:
Created a pull request template in `.github`.

Is there anything else you would like to have in the template?
Should we move `CONTRIBUTING.md` to the `.github` directory as well?

miketaylr commented:
> Should we move CONTRIBUTING.md to the .github directory as well?

I opened https://github.com/webcompat/webcompat.com/issues/2192 to discuss moving over all docs there. CONTRIBUTING.md was the least controversion, so you could add a commit for that in this PR!

miketaylr commented on .github/pull_request_template.md:
```diff
@@ -0,0 +1,8 @@
+By submitting this pull request, I promise that:
+
+- [ ] I have read the [Pull Request + Code Style Guidelines](https://github.com/webcompat/webcompat.com/blob/master/docs/pr-coding-guidelines.md) thoroughly.
+- [ ] I ensure my pull request follows each and every point.
```
I'm not sure about this second line, is this a common pattern other projects follow?

karlcow replied:
Drive-by comment:
The template if I understood will be displayed each time someone create a pull request. It will become monotonous for people contributing often to have to this message.

On the other hand it could be useful to have in the template something guiding the person to improve the pull request.

```md
This PR fixes issue `#nnnn`

## Proposed PR background
(put here comments helping us to understand the PR.)

PS: Make sure your PR is following [Pull Request + Code Style Guidelines](https://github.com/webcompat/webcompat.com/blob/master/docs/pr-coding-guidelines.md)
```

mischah replied:
> I'm not sure about this second line, is this a common pattern other projects follow? 

I just it in a Repo where people continuously don’t stick to the rules. Removed it with the latest commit.

karlcow commented:
Some examples https://github.com/search?q=pull+request+template&type=Issues

mischah commented:
Here a few template templates which we could adapt as well:
https://github.com/stevemao/github-issue-templates

mischah commented:
### Update

Updated the PR-Template with 9deab0b

> I opened #2192 to discuss moving over all docs there. CONTRIBUTING.md was the least controversion, so you could add a commit for that in this PR!

Moved CONTRIBUTING.md to .github directory with 1ecc092

Let me know if you want any updates to the PR template or if you want me to squash the commits.

mischah commented:
I guess this could need another round of review 😊 

r? @miketaylr @karlcow @zoepage

mischah commented:
👌
